dags/remove_old_processor_logs.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+"""
+# DAG to remove dag_processor_log files older than 10 days.
+"""
+from airflow import DAG
+from datetime import datetime, timedelta
+from airflow.operators.python_operator import PythonOperator
+from dags.exec_in_pod import exec_in_pod
+
+import os
+
+START_DATE = datetime.now() - timedelta(days=2)
```

Comment:
Because your schedule interval is `@weekly`, I think that you want the `START_DATE` to be two weeks before the current date. See https://airflow.apache.org/docs/stable/scheduler.html